core/src/test/java/org/hibernate/ogm/utils/AvailableGridDialect.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.utils;
+
+
+/**
+ * The testsuite needs some knowledge on all NoSQL dialects.
+ * We mainly need the name of the GridDialect class.
+ *
+ * @author Mark Paluch
+ */
+public enum AvailableGridDialect {
```

Comment:
The enum is not necessary, but comes handy, you don't have to remember the FQCN all the time. It follows the `GridModule` (former `GridDialectType`) and `AvailableDatastoreProvider` style. 

Referencing the class is not possible because the tests are located in `core` and have no dependencies to the grid modules.